NEWS

```diff
@@ -1,3 +1,15 @@
+libmongoc 1.25.3
+================
+
+Fixes:
+
+  * Fix data race in `mongoc_cursor_get_host`.
```

Comment:
```suggestion
  * Disable shared libmongoc targets if `ENABLE_SHARED=OFF`
```

To match wording in: https://github.com/mongodb/mongo-c-driver/releases/tag/1.25.3

The data race fix was included in 1.25.2. Suggested change is to include CDRIVER-4784,